hardhat.config.js

```diff
@@ -71,6 +71,11 @@ module.exports = {
 			chainId: 1,
 			accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
 		},
+		harmony: {
+			url: 'https://a.api.s0.t.hmny.io',
```

Comment:
Sometimes I saw a bit of read-delay (by a few blocks) from this node. Probably should still keep it overridable by env